src/dialogs/more-info/controls/more-info-media_player.ts

```diff
@@ -62,7 +62,8 @@ class MoreInfoMediaPlayer extends LitElement {
                 `
               )}
         </div>
-        ${supportsFeature(stateObj, MediaPlayerEntityFeature.BROWSE_MEDIA)
+        ${stateActive(stateObj) &&
```

Comment:
Is `stateActive` really the correct filter? Does that mean you can't browse in `standby`?

`!isUnavailableState` instead maybe?

Reply:
I initially had used `isUnavailableState`. If browsing in standby is a valid supported use case, then reverting back to `isUnavailableState` sounds reasonable.

Reply:
yeah I agree `isUnavailableState` is the correct check